sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -57,10 +57,10 @@ def formation(B, ds, radius, k):
     for ac in list_aircraft:
         waiting_for_msgs = 0
         if ac.a == -999:
-            print "Waiting for GVF msg of aircraft ", ac.id
+            print ("Waiting for GVF msg of aircraft ", ac.id)
```

Comment:
in python you usually don't put a space between the function name and the opening bracket...